apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -172,7 +172,7 @@
       "inputValue": {
         "appContentUrl": "www.someUrl.com"
       },
-      "expectedAlertValue": "shareAppContentToStage called",
+      "expectedAlertValue": "shareAppContentToStage called with [object Object] + [object Object]",
```

Comment:
This makes me sad and happy because it highlights that we probably have some parsing to do to actually validate the results (i.e., stringify them).

Reply:
Yeah, there are definitely some things we should clean up in here but for now I focused on making things work as-is